### PR TITLE
[v8.3.x] JWT: Split race-y test into two stable tests

### PR DESCRIPTION
**Pull request by grafanabot**
Backport 12e0a94316b2d310e44426196c083fb3f098c086 from #41950

### CODE REVIEW

**Review by sakjur — APPROVED** [review]

**Comment by sakjur** [comment]
@dsotirakis Is it safe to merge this today or should I wait until after the beta release?

**Comment by dsotirakis** [comment]
@sakjur, I think we **need** to merge that today, since it failed for a couple of PRs as I've seen it. Better safe than sorry!

**Pull request merged**
